testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/InternationalizationTest.java

```diff
@@ -93,6 +105,34 @@ public void userAttributeTest() {
         assertCustomLocalePersonalInfo();
     }
 
+    @Test
+    public void shouldDisplayTimeUsingSelectedLocale() {
+        signingInPage.navigateTo();
+        loginToAccount();
+        SigningInPage.UserCredential passwordCred =
+                passwordCredentialType.getUserCredential(testUserResource().credentials().get(0).getId());
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMMM d, yyyy, h:mm a");
```

Comment:
Locale is not specified in the formatter. Is that intentional? I know it defaults to English in most cases but not sure what happens if someone runs the test on a machine with different locale.